classes/class-extensions.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace HM\BackUpWordPress;
+
+class Extensions {
```

Comment:
Ideally we'd:

- Docblock the class and it's methods
- Add unit tests for it's functionality
- Follow the Singleton pattern set by `Path` and `Notices`